podcasts/ProfileIntroViewController.swift

```diff
@@ -199,8 +207,14 @@ extension ProfileIntroViewController: ASAuthorizationControllerDelegate {
     func showError(_ error: Error) {
         FileLog.shared.addMessage("Failed to connect SSO account: \(error.localizedDescription)")
         let error = (error as? APIError) ?? .UNKNOWN
-        Analytics.track(.userSignInFailed, properties: ["source": AuthenticationSource.ssoApple.rawValue, "error_code": error.rawValue])
+        var message = L10n.accountSsoFailed
+        if error != .UNKNOWN, !error.localizedDescription.isEmpty {
+            message = error.localizedDescription
```

Comment:
I wonder if we should show specific errors on the interface. The regular user probably wouldn't understand it, so a generic "Sign in with Apple failed. Please try again." is better IMHO.

Reply:
Sounds good. I was looking to maintain consistency with the other screens, but I'm happy to keep it fully generic.